controllers/admin/AdminImportController.php

```diff
@@ -1939,13 +1939,15 @@ protected function productImportOne($info, $default_language_id, $id_lang, $forc
         }
 
         // Category default now takes the value of the first new category during import
-        if (isset($product->id_category[0])) {
-            $product->id_category_default = (int) $product->id_category[0];
-        } else {
-            if (!isset($product->id_category_default) || !$product->id_category_default) {
+        if (!isset($product->id_category_default) || !$product->id_category_default) {
```

Comment:
Isn't bit a little weird, as I understand this means we can import product data and update the default category if it already has one :thinking: ?

Reply:
In fact, what this mean is: we cannot assign a new default category to a product that already has a defined default category and no default category modification has been requested on the CSV import. Only if an specific default category update is requested the change will be made (on the fillInfo method, previous to this code) and if default category is not present on id_category property, then it will be added.